python_modules/dagster-graphql/dagster_graphql/implementation/execution/run_lifecycle.py

```diff
@@ -39,7 +39,12 @@ def is_resume_retry(execution_params):
 
 
 def create_valid_pipeline_run(graphene_info, external_pipeline, execution_params):
-    ensure_valid_config(external_pipeline, execution_params.mode, execution_params.run_config)
+    if execution_params.mode is None and len(external_pipeline.available_modes) == 1:
```

Comment:
My understanding was that, in other execution situations, we raise an error when no mode is supplied and there's no mode named default. Is that incorrect? If it is the case, it would be preferable to be consistent.